code/game/objects/structures.dm

```diff
@@ -4,6 +4,7 @@
 	var/climbable
 	var/mob/climber
 	var/broken = FALSE
+	#define isstructure(A)	(istype((A), /obj/structure))

```

Comment:
Ah didn't realize the file path I was in, I was in _Defines mobs.dm . By the way why are mobs defined in the mobs.dm instead of is_helpers.dm? Were they created before is_helpers became a file of its own?